game/scripts/vscripts/items/meteor_hammer.lua

```diff
@@ -0,0 +1,203 @@
+LinkLuaModifier("modifier_generic_bonus", "modifiers/modifier_generic_bonus.lua", LUA_MODIFIER_MOTION_NONE)
+LinkLuaModifier("modifier_item_meteor_hammer_thinker", "items/meteor_hammer.lua", LUA_MODIFIER_MOTION_NONE )
+LinkLuaModifier("modifier_item_meteor_hammer_damage_over_time", "items/meteor_hammer.lua", LUA_MODIFIER_MOTION_NONE)
+
+item_meteor_hammer = class(ItemBaseClass)
+item_meteor_hammer_1 = item_meteor_hammer
+item_meteor_hammer_2 = item_meteor_hammer_1
+item_meteor_hammer_3 = item_meteor_hammer_1
+item_meteor_hammer_4 = item_meteor_hammer_1
+item_meteor_hammer_5 = item_meteor_hammer_1
+
+function item_meteor_hammer:OnSpellStart()
+
+  local caster = self:GetCaster()
+
+  caster:EmitSound("DOTA_Item.MeteorHammer.Channel")
+
+  caster:StartGesture(ACT_DOTA_TELEPORT)
+
+  if IsServer() then
+    --dehardcode ---------------------------------------
```

Comment:
If you meant to de-hardcode the vision time, then I suggest you do that by making a KV for it, or whatever else you might have thought of. Otherwise, you might want to remove the comment or make it more detailed so that it isn't confusing later on.

Reply:
I forgot to remove it, there was a piece of code that was hardcoded there before.